project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 // Comment to get more information during initialization
 logLevel := Level.Warn
 
-resolvers += "Typesafe repository" at "http://repo.typesafe.com/typesafe/releases/"
+resolvers += "Typesafe repository" at "https://repo.typesafe.com/typesafe/releases/"
```

Comment:
this was causing a warning because http repos could be hijacked and bad stuff injected.  If we go straight to https then that takes away the opportunity to be redirected somewhere bad.